libs/portals/admin/form-system/src/components/FormLayout/components/MainColumns.tsx

```
@@ -94,7 +95,7 @@
             onConfirm={remove}
             disclosureElement={
               <DeleteButton
-                onClick={() => {}}
+                onClick={() => { }}
```

ESLint

[error] Unexpected empty arrow function.

(@typescript-eslint/no-empty-function)